pulsar-broker-common/src/main/java/org/apache/pulsar/broker/cache/ConfigurationCacheService.java

```diff
@@ -101,6 +104,13 @@ public Policies deserialize(String path, byte[] content) throws Exception {
             }
         };
 
+        this.resourcegroupsCache = new ZooKeeperDataCache<ResourceGroup>(cache) {
```

Comment:
Since we're moving towards having all accesses through the metadata store API, is there a way to avoid this?

Reply:
Thanks for catching this, removed it.